the-telephone-app-fe/src/components/StartGame/StartGame.js

```diff
@@ -1,7 +1,62 @@
+import React, { useState } from 'react';
+import axios from 'axios';
+import { useNavigate } from 'react-router-dom';
 import './StartGame.css';
 import imgLogo from '../../assets/gartic-phone.svg';
 import { BsFillCaretRightFill } from "react-icons/bs";
+import {over} from 'stompjs';
+import SockJS from "sockjs-client";
+import { useParams } from 'react-router-dom';
 const StartGame = () => {
+    const [name, setName] = useState('Nickname' + Math.round(Math.random() * 100000) );
+   //lấy id room <-- link
+    const handleNameChange = (e) => {
+        // setName(e.target.value);
+        if(e.target.value) {
+            setName(e.target.value);
+        }
+        else {
+            setName('Nickname' + Math.round(Math.random() * 1000));
+        }
+    }
+    const navigate = useNavigate();
+    const onError =(err)=>{
+        console.log(err);
+    }
+    const { id_room } = useParams();
+    console.log(id_room);
+    var client = null;
+    const onConnected = (id_room,data,host) => {
+      
+        client.subscribe('/topic/'+id_room,
+        function (response) {
+             data = JSON.parse(response.body);
+             navigate('/lobby', { state: { data, id_room,  host} });
+        }
+    
+    );
+    navigate('/lobby', { state: { data,id_room ,host} });
+   
+    }
+    
+    const handleStartClick = async ()=> {
+        const response = await axios.post(`http://192.168.101.43:9090/user/create/${name}`);
```

Comment:
Define the API URL 'http://192.168.101.43:9090' as constant in a config file to reuse and update easily, as this was used many times and will change in the future.